neqo-http3/tests/priority.rs

```diff
@@ -68,7 +68,7 @@ fn priority_update() {
             Instant::now(),
             "GET",
             &("https", "something.com", "/"),
-            &[],
+            &[Header::new("priority", "u=4,i")],
```

Comment:
Is the test verifying that this gets used somewhere?

Reply:
Yes, line 96 in expected_headers.